googleapiclient/http.py

```diff
@@ -19,22 +19,21 @@
 actual HTTP request.
 """
 from __future__ import absolute_import
-import six
 
 __author__ = "jcgregorio@google.com (Joe Gregorio)"
 
-from six import BytesIO, StringIO
-from six.moves.urllib.parse import urlparse, urlunparse, quote, unquote
-
 import copy
 import httplib2
+import http.client as http_client
```

Comment:
Same here:
```suggestion
import http.client
```

Reply:
See my comment above regarding the naming conflict with `http` here https://github.com/googleapis/google-api-python-client/blob/master/googleapiclient/http.py#L901